lib/entities/entry.ts

```diff
@@ -353,6 +365,20 @@ function createEntryApi(makeRequest: MakeRequest): EntryApi {
       const raw = this.toPlainObject() as EntryProps
       return checks.isArchived(raw)
     },
+
+    references: function references(maxDepth: number) {
```

Comment:
you could probably document this as well, e.g.:

```
const testEntry = await environment.getEntry('<entry_id>')
const references = await testEntry.references(2)

// or

environment.getEntry('<entry_id>').then((entry) => entry.references(2) )

```

Reply:
done